lib/gui/models/selection-state.js

```diff
@@ -23,12 +23,11 @@
 const _ = require('lodash');
 const angular = require('angular');
 const Store = require('./store');
+const availableDrives = require('./drives');
```

Comment:
Can we rename the file to `available-drives.js`, so it better matches the name we're giving it here?